airflow/providers/microsoft/azure/hooks/wasb.py

```diff
@@ -134,7 +134,7 @@ def get_conn(self) -> BlobServiceClient:
             app_id = conn.login
             app_secret = conn.password
             tenant = extra.get('tenant_id') or extra.get('extra__wasb__tenant_id')
-            token_credential = ClientSecretCredential(tenant, app_id, app_secret)
+            token_credential = ClientSecretCredential(tenant, app_id, app_secret)  # type: ignore
```

Comment:
Same

Reply:
The error here was:
```
airflow/providers/microsoft/azure/hooks/wasb.py:137: error: Argument 1 to "ClientSecretCredential" has incompatible type "Optional[Any]"; expected "str"
                token_credential = ClientSecretCredential(tenant, app_id, app_secret)
```
I'll address this properly.